Sail/src/Sail/entities/components/CollisionComponent.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "Component.h"
+#include "..//..//Physics/Octree.h"
+
+class CollisionComponent : public Component<CollisionComponent> {
+public:
+	CollisionComponent() {}
+	~CollisionComponent() {}
+
+	float airDrag = 1.0f;
```

Comment:
Air drag only possible for entities that can collide? Maybe move this to MovementPostCollisionSystem?

Reply:
Moved airDrag to from CollisionComponent to MovementComponent and moved code from CollisionSystem to MovementSystem.